pkg/command/context.go

```diff
@@ -273,9 +274,13 @@ func createCtx(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
+	// TODO: update the below conditional check (and in login command) after context scope plugin support
+	//       is implemented for tanzu context(TAP SaaS)
 	// Sync all required plugins
-	if err := syncContextPlugins(cmd, ctx.ContextType, ctxName); err != nil {
-		log.Warningf("unable to automatically sync the plugins recommended by the new context. Please run 'tanzu plugin sync' to sync plugins manually, error: '%v'", err.Error())
+	if ctx.ContextType != configtypes.ContextTypeTanzu {
```

Comment:
We should probably add the same check under the `useCtx` function as well to not sync plugins for the `tanzu` context.

Reply:
Just noticed it, was about to update the PR with other comments. Thanks